metaci/release/utils.py

```diff
@@ -78,26 +83,28 @@ def update_release_from_github(release, repo_api=None):
     return release
 
 
-def send_release_webhook(project_config, release, config_item=None):
-    if release is None or not settings.METACI_RELEASE_WEBHOOK_URL:
+def send_release_webhook(release, config_item=None):
+    if (
+        release is None
+        or not settings.METACI_RELEASE_WEBHOOK_URL
+        or not settings.GUS_BUS_OWNER_ID
```

Comment:
GUS_BUS_OWNER_ID is required in order for us to send implementation steps, but it is not required for us to create a change case with no implementation steps. I don't think we want it included in the check here.